wordpress-mobile/WordPress-Android
# Fix Media Browser message after adding or removing a photo

Fixes #7304 

To test:

**Test 1**
1. Open empty media library.
2. From the add menu at the top right, take a photo.
3. Make sure photo is added to the media library.
4. "You don't have any images" is no more visible.

<img src="https://user-images.githubusercontent.com/2581647/36751258-6de19a1e-1bde-11e8-9a5c-8ada9020e74d.png" width="200">

**Test 2**
1. Open media library.
2. Click on the photo taken in **test 1** to show File Details
3. Delete the photo 
4. The screen is empty with the message:"You don't have any images."

<img src="https://user-images.githubusercontent.com/2581647/36751268-7422850a-1bde-11e8-9351-b0d1adf689e6.png" width="200">

Tested on Nexus 5X API 25 emulator.

cc @khaykov @theck13
